lib/widgets/card_content.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:flutter/material.dart';
+
+class CardContent extends StatelessWidget {
+  final String title;
+  final String fullName;
+  final String text;
+  final double horizontalPadding;
+  final double fontSize;
+  final double iconsSize;
+
+  const CardContent(
+      {Key? key,
+      this.title = 'Document',
+      this.fullName = 'Lorem Ipsum',
+      this.text =
+          'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.',

```

Comment:
it's better to extract long text like this to some constant (declared in this file outside/inside of `CardContent` or in separate file if it's used in multiple places)

Reply:
Fixed.